aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1695,14 +1695,17 @@ Tensor sparse_compressed_to_sparse_csc(const Tensor& self, c10::optional<int64_t
   return self;
 }
 
+Tensor sparse_coo_to_sparse(const Tensor& self, const int64_t sparse_dim) {
+  TORCH_CHECK(
+      sparse_dim >= 0 && sparse_dim <= self.sparse_dim(), "sparse_dim argument for sparse_coo_to_sparse must be between 0 and number of sparse dimensions of original tensor");
```

Comment:
Isn't this check a bit redundant? The user will be really angry to hit the first, and then, after potentially a couple of ours, the second :)

Reply:
Removed (this PR initially proposed the second check as warning instead).